src/Logging/Logging.Console/Extensions/LogService.Console.Extensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace CodeMonkeys.Logging.Console
+{
+    public static partial class LogServiceExtensions
+    {
+        /// <summary>
+        /// If a <see cref="ConsoleLogService"/> is attached to the <see cref="LogService"/> this method enables
+        /// logging with it.
+        /// </summary>
+        public static void EnableConsoleLogging(this ILogService @this)
+        {
+            Argument.NotNull(
+                @this,
+                nameof(@this));
+
+            @this.EnableScopedService<ConsoleLogService>();
+        }
+
+
+        /// <summary>
+        /// If a <see cref="ConsoleLogService"/> is attached to the <see cref="LogService"/> this method disables
+        /// logging with it.
+        /// </summary>
+        public static void DisableConsoleLogging(this ILogService @this)
+        {
+            Argument.NotNull(
+                @this,
+                nameof(@this));
+
+            @this.DisableScopedService<ConsoleLogService>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'this' before using it.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)